### PR TITLE
Fixes for WebGPU + wasm64

### PR DESCRIPTION
**Pull request by sbc100**
These are enough to get the compile and link tests to work.

Fixes: #19866

### CODE REVIEW

File: src/library.js [hunk 1]
```diff
@@ -2736,6 +2736,7 @@ mergeInto(LibraryManager.library, {
   emscripten_pc_get_function: (pc) => {
 #if !USE_OFFSET_CONVERTER
     abort('Cannot use emscripten_pc_get_function without -sUSE_OFFSET_CONVERTER');
+    return 0;
```
Comment (sbc100) [inline src/library.js:2739]: Without this I was setting a closure warning/error when attempting to convert the return value of this function to a BigInt.

File: src/library_async.js [hunk 2]
```diff
@@ -657,12 +657,6 @@ mergeInto(LibraryManager.library, {
   emscripten_scan_registers: function(func) {
     throw 'Please compile your program with async support in order to use asynchronous operations like emscripten_scan_registers';
   },
-  emscripten_fiber_init: function() {
-    throw 'Please compile your program with async support in order to use asynchronous operations like emscripten_fiber_init';
-  },
-  emscripten_fiber_init_from_current_context: function() {
-    throw 'Please compile your program with async support in order to use asynchronous operations like emscripten_fiber_init_from_current_context';
-  },
```
Comment (sbc100) [inline src/library_async.js:665]: These two functions are now implement in native code and never undefined, so these fallbacks are no longer needed.

File: src/library_webgpu.js [hunk 3]
```diff
@@ -1491,7 +1491,7 @@ var LibraryWebGPU = {
   wgpuQueueOnSubmittedWorkDone: function(queueId, signalValue, callback, userdata) {
     var queue = WebGPU.mgrQueue.get(queueId);
 #if ASSERTIONS
-    assert(signalValue_low === 0 && signalValue_high === 0, 'signalValue not supported, must be 0');
+    assert(signalValue === 0, 'signalValue not supported, must be 0');
```
Comment (kainino0x) [inline src/library_webgpu.js:1494]: If `signalValue` can be a BigInt then this should be `==`
```suggestion
    assert(signalValue == 0, 'signalValue not supported, must be 0');
```
(I'm sure this won't be the only bug, but I spotted it now, so may as well fix it 🙂)

File: src/library_webgpu.js [hunk 4]
```diff
@@ -1759,7 +1759,7 @@ var LibraryWebGPU = {
     encoder["insertDebugMarker"](UTF8ToString(markerLabelPtr));
   },
 
-  wgpuCommandEncoderFinish: function(encoderId) {
+  wgpuCommandEncoderFinish: function(encoderId, descriptor) {
```
Comment (kainino0x) [inline src/library_webgpu.js:1762]:
```suggestion
  wgpuCommandEncoderFinish: function(encoderId, descriptor) {
    // TODO: Use the descriptor.
```

**Review by kainino0x — APPROVED** [review]
LGTM with small suggestions

**Pull request merged**